CRM/Contribute/Form/AbstractEditPayment.php

```diff
@@ -391,9 +391,9 @@ public function assignProcessors() {
       // for some reason there was a need to filter here per commit history - but this indicates a problem
       // somewhere else.
       if ($processor['is_test'] == ($this->_mode == 'test')) {
-        $this->_processors[$id] = ts($processor['name']);
+        $this->_processors[$id] = $processor['name'];
         if (!empty($processor['description'])) {
-          $this->_processors[$id] .= ' : ' . ts($processor['description']);
+          $this->_processors[$id] .= ' : ' . $processor['description'];
```

Comment:
Again, these are things that may work in some circumstances, but only by chance. It should be fixed at the source.